### PR TITLE
Application

### PR DESCRIPTION
**Pull request by acBerger**
I tried to write out the points in "Application" (Chapter 2).
I am not very happy with my results for 2.1 (Application Fields) as I think we should add some more information here.
So please tell me, if you know something to add.
I created this pull request as I think even through it is does not finish #66 yet, it improves our current SRS.

### CODE REVIEW

**Comment by viridispirus** [comment]
lgtm.

File: doc/Requirements Specification/Parts/Application.lyx [hunk 1]
```diff
@@ -104,7 +104,49 @@ name "/A10/"
...
+\begin_layout Plain Layout
+
+
+\backslash
+Gls{Palladio}
+\end_layout
+
+\end_inset
+
+ for a existing software Beagle is helpful to detect bottlenecks and problems
```
Comment (jGleitz) [inline doc/Requirements Specification/Parts/Application.lyx]: a[n] existing software

Comment (RomanLangrehr) [inline doc/Requirements Specification/Parts/Application.lyx]: I think there should not be an "an".

Comment (jGleitz) [inline doc/Requirements Specification/Parts/Application.lyx]: it’s an “a” before a vocal, of course there should be a “n”!

Comment (RomanLangrehr) [inline doc/Requirements Specification/Parts/Application.lyx]: Yes, sorry.
I thougth you were referencing to line 107, and there should be no a/an at all.

**Comment by jGleitz** [comment]
lgtm

**Pull request merged**